tests/test_br.py

```diff
@@ -177,3 +178,30 @@ def test_BRStateChoiceField(self):
             'pr': error_invalid,
         }
         self.assertFieldOutput(BRStateChoiceField, valid, invalid)
+
+
+class BRLocalFlavorModelTests(SimpleTestCase):
```

Comment:
You should probably also check that the correct validator is set on the model fields just to make sure it's the correct one.